.github/ISSUE_TEMPLATE/data_request.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Data request
+about: Suggest data to add to this platform
+
+---
+
+**Please describe the data required**
```

Comment:
Just checking here, don't we need a WHY is the data needed or by the time we get the request this would have already be decided?

Reply:
@kilemensi maybe I could add it to this section, `describe the data required and why it's required?`